docs/setup/install.md

```diff
@@ -0,0 +1,29 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
```

Comment:
```suggestion
- A Kubernetes cluster with version 1.21 or newer is running and the SuspendJob feature gate is enabled.
```

Reply:
on version?

Reply:
```suggestion
- A Kubernetes cluster with version 1.21 or newer is running.
- The SuspendJob feature gate is enabled.
```
Based on @alculquicondor suggestion. Assuming we could split the latter requirement bullet.

Reply:
Maybe we can be more specific.

```
The SuspendJob feature gate is enabled. In Kubernetes 1.22 or newer, the feature gate is enabled by default.
```

Then we can remove the paragraph below.